language/Portuguese_Brazil/strings.po

```diff
@@ -2260,11 +2269,11 @@ msgstr "Pré-escala de Bitmap"
 
 #: src/frontend/mame/ui/submenu.cpp:109
 msgid "Window Mode"
-msgstr "Modo Janela"
+msgstr "Modo de Janela"
```

Comment:
I'm pretty sure the previous translation is better here - this is for selecting a mode where MAME runs windowed, not selecting the mode of a window.

Reply:
I disagree. "Modo Janela" is better here indeed. "Modo Janelado" could work as well, but I think it may sound a bit weird.